### ISSUE TITLE
define spec2vec_similarity throws AttributeError

### ISSUE THREAD

**Issue by chrispook** [iomega/spec2vec#67]
Hi, I've got as far as the step on calculating spectrum similarities in the demo on the NL eSceinces Centre blog but when I try to run:
_spec2vec_similarity = Spec2Vec(model=model, intensity_weighting_power=0.5, allowed_missing_percentage=5.0)_

I get _AttributeError: 'Word2VecKeyedVectors' object has no attribute 'key_to_index'_

Can you suggest a fix? I'm running Python 3.8 in Jupyter-notebook and the latest versions of matchms and spec2vec. Everything's worked perfectly up til here.
Thanks

**Comment by florian-huber** [comment]
Thank for posting this issue @chrispook.
In the coming days I will see if I can reproduce it and find the cause. Looks like an issue with `gensim`, maybe related to https://github.com/RaRe-Technologies/gensim/issues/1994

Can you check what gensim version you have (e.g. running `!pip list` from the notebook) ?

**Comment by chrispook** [comment]
Hi Florian, thank you for your fast response. Apparently I have gensim 3.8.3. I have spent the last four hours trying to update to v4.01. and getting nowhere. There seem to be a lot of incompatibilities with my current install. I am now trying it on my Linux laptop. I will report back. 
Chris

**Comment by florian-huber** [comment]
Can be quite annoying those incompatibility issues 🙄 .

I would hope that it would work with a making a fresh environment (if you're using conda or similar). If that won't work, another patch could be to downgrade Spec2Vec to version 0.4.0?

**Comment by chrispook** [comment]
I also had endless problems with Linux  so I went back to Windows and updated it with pip instead of conda. This may yield incompatibilities elsewhere but it's working now. One odd thing I had to change was to make the line read _spec2vec_similarity = **Spec2Vec.Spec2Vec**(model=model, intensity_weighting_power=0.5, allowed_missing_percentage=5.0, 
                                   progress_bar=True)_
Hope this helps. Thanks for the great toolset. Looking forward to getting results.
